### ISSUE TITLE
Length limit changed, finally!!

### ISSUE THREAD

**Issue by Anonymous941** [ShadowOne333/MaternalBound-Redux#13]
(I'll edit more details in later)

When working on #12, I remembered about how I managed to change the name length easily with the disassembly ebsrc!  [Here's a simple test patch to demonstrate it](https://github.com/user-attachments/files/16046593/earthbound-name-change.zip) ([source](https://github.com/Anonymous941/ebsrc/tree/name-change)) (apply on a headered ROM).  Don't care and the window lengths are not working, but I got them working in the past, so I can change them when I get more time.  I don't really deserve credit for this, as ebsrc had most of the work done for me already

It will still need some work to integrate into a CoilSnake project like this, so I can work on that when I have some time

**Comment by Anonymous941** [comment]
Virtually unlimited, I think I got it working with 15 or 20

**Comment by ShadowOne333** [comment]
This looks quite promising.
If you have the time, and are willing to, please consider joining the EBB Remake server, the developer for the Remake is quite interested in this. It'd also be of great help for EarthBound hacking as a whole.

Here's the invite to the server:
https://discord.gg/N8TTRNGb

**Comment by Anonymous941** [comment]
@ShadowOne333 Absolutely, I'd be glad to help.  I'm currently working on another project, but I should have some time for this

**Comment by ShadowOne333** [comment]
I think the whole licensing stuff is not an issue, given how much of the code, if not the entirety of it, for the Extended Name Length hack would have to be rewritten for CoilSnake at the end of the day.

**Comment by Anonymous941** [comment]
Just curious, was this functionality added?

**Comment by ShadowOne333** [comment]
Only the six letter patch was added in.
Sadly there's no plans to implement further length text for the names given how it can cause issues with the HP/PP boxes.
The current 6 letter patch works just perfectly with those.
